doc/workflows/anaconda-webui.rst

```diff
@@ -0,0 +1,161 @@
+- **debug** - If true VMs is not removed at the end of the test
+
+VMHypervisors
+^^^^^^^^^^^^^
+Hypervisor hostnames or IPs, one for each supported architecture.
+Use `localhost` for local hypervisor.
+
+AnacondaWebUIRepos
+^^^^^^^^^^^^^^^^^^
+Dictionary of git repositories with additional test scripts.
+
+AnacondaWebUIkernelCmdline
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+Additional arguments for kernel cmdline, one for each supported architecture and
+one added to all architectures.
+
+Usage
+-----
+
+Testcase
+^^^^^^^^
+Execution type of this workflow is ``anaconda-webui``. Required automation_data
+are:
+
+- **script_file** - path to the test script, by default the base for this path
+  is anaconda git repository
+- **test_case** - python class in the script file (test script file can contain 
+  multiple test cases)
+
+Optional automation_data:
+
+- **test_repo** - Name of repository where the test script is located. Repositories
+  are defined in AnacondaWebUIRepos settings section. Sets the base path for script_file.
+- **additional_repos** - List of additional repos for installation (``inst.addrepo``), eg. ``['AppStream', 'CRB']``.
+  These need to be supplied by InstallationSource event structure.
+- **kernel_cmdline** - Parameters to be added to kernel cmdline.
+- **webui_startup_timeout** - Override how long to wait for WebUI to start (in minutes).
+
+Example of minimal execution section for running test from anaconda repo::
+
+    execution:
+        type: anaconda-webui
+        automation_data:
+            script_file: ./ui/webui/test/integration/default.py
+            test_case: DefaultInstallation
+
+Example of execution section with all options set::
+
+    execution:
+        type: anaconda-webui
+        automation_data:
+            script_file: ./check-navigation.py
+            test_case: TestNavigation
+            test_repo: my-special-tests
+            additional_repos: ['AppStream', 'CRB']
+            kernel_cmdline: 'nosmt'
+            webui_startup_timeout: 15
+
+Hypervisor
+^^^^^^^^^^
+The workflow is using virtual machines, so it needs access to a system with libvirtd running.
+That can be eighter localhost, in that case the user that starts Permian needs to be in the
+group `libvirt`. Or any remote system with password-less access to root account via SSH.
+This is required because the workflow is using iptables to setup port prerouting.
+
+For prerouting the workflow is using ports 42000+. Idealy the remote hypervisor should not
+be used for anything else to avoid issues.
+
+Test scripts
+^^^^^^^^^^^^
+Anaconda WebUI integration tests are python scripts that use cockpit's test framework
+and wrapper methods that make it easy to interact with the Web UI and run commands
+on the machine during installation and after reboot.
+
+More information can be found in `Anaconda documentation <https://anaconda-installer.readthedocs.io/en/latest/testing.html#anaconda-web-ui-tests>`_.
```

Comment:
The link doesn't work yet, see https://github.com/rhinstaller/anaconda/pull/4667